api/webhook.ts

```diff
@@ -31,61 +24,10 @@ webhookRoutes.get("/", (req, res) => {
   }
 });
 
-webhookRoutes.post("/", async (req, res) => {
-  // log incoming messages
-  console.log("Incoming webhook message:", JSON.stringify(req.body));
-
-  // check if the webhook request contains a message
-  // details on WhatsApp text message payload: https://developers.facebook.com/docs/whatsapp/cloud-api/webhooks/payload-examples#text-messages
-  const message = req.body.entry?.[0]?.changes[0]?.value?.messages?.[0];
-
-  // check if the incoming message contains text
-  if (!message?.type) {
-    res.sendStatus(200);
-    return;
-  }
-
-  // aknowledge that the message has been read and be processed
-  await markChatAsRead(message.id);
-
-  let chatbotReply = null;
-  let queryText = "";
-
-  switch (message.type) {
-    case "interactive":
-      if (message.interactive.type === "button_reply") {
-        queryText = message.interactive.button_reply.id;
-      } else if (message.interactive.type === "list_reply") {
-        queryText = message.interactive.list_reply.id;
-      } else if (message.interactive.type === "nfm_reply") {
-        queryText = "";
-      }
-      break;
-    default:
-      queryText = message.text.body;
-      break;
-  }
-
-  if (!queryText) {
-    res.sendStatus(200);
-    return;
-  }
-
-  if (CONNECTION_PLATFORM === DIFY) {
-    chatbotReply = await queryToDify({ req, query: queryText });
```

Comment:
this PR makes too much changes. The logic should be like this:

1. webhook endpoint receive request
2. send `markChatAsRead` to queue
3. send `queryToDify` to queue
4. return status 200

while in parrallel a consumer:
1. do `markChatAsRead`

other consumer, do in 1 action:
1. do `queryToDify`
2. receive the query return
3. do `sendChatbotReply`